spec/mailers/user_mailer_spec.rb

```diff
@@ -533,138 +533,193 @@ def expect_email_body_to_have_help_and_contact_links
     end
   end
 
-  describe '#in_person_ready_to_verify' do
+  context 'in person emails' do
+    let(:capture_secondary_id_enabled) { false }
+    let(:current_address_matches_id) { false }
     let!(:enrollment) do
       create(
         :in_person_enrollment,
         :pending,
         selected_location_details: { name: 'FRIENDSHIP' },
         status_updated_at: Time.zone.now - 2.hours,
+        capture_secondary_id_enabled: capture_secondary_id_enabled,
+        current_address_matches_id: current_address_matches_id,
       )
     end
+    describe '#in_person_ready_to_verify' do
+        let(:mail) do
+        UserMailer.with(user: user, email_address: email_address).in_person_ready_to_verify(
+          enrollment: enrollment,
+        )
+      end
 
-    let(:mail) do
-      UserMailer.with(user: user, email_address: email_address).in_person_ready_to_verify(
-        enrollment: enrollment,
-      )
-    end
+      it_behaves_like 'a system email'
+      it_behaves_like 'an email that respects user email locale preference'
 
-    it_behaves_like 'a system email'
-    it_behaves_like 'an email that respects user email locale preference'
-  end
+      context 'double address verification is not enabled' do
+        it 'renders the body' do
+          expect(mail.html_part.body).
+            to have_content(
+              t('in_person_proofing.process.proof_of_address.heading'),
+            )
```

Comment:
It might be overkill but what do you think about adding  a check if in_person_proofing.process.proof_of_address.info exists or does not? It has the same behavior as heading so maybe not necessary.